guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -92,6 +110,12 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
         // that indicate the clipboard field may have been changed
         element.addEventListener('input', updateClipboardData);
         element.addEventListener('change', updateClipboardData);
+        element.addEventListener('focus', displayAndUpdateClipboardEditor);
+
+        // Update remote clipboard if local clipboard changes
```

Comment:
Is this copy pasta? Looks like it's a verbatim copy of the block directly below.

Reply:
Whoops. Overlooked this when checking the diff before I pulled